website/pages/api-docs/system/quotas-config.mdx

```diff
@@ -0,0 +1,67 @@
+---
+layout: api
+page_title: /sys/quotas/config- HTTP API
+sidebar_title: <code>/sys/quotas/config</code>
+description: The `/sys/quotas/config` endpoint is used to configure rate limit quotas.
+---
+
+# `/sys/quotas/config`
+
+The `/sys/quotas/config` endpoint is used to configure rate limit quotas.
+
+## Create or Update the Rate Limit Configuration
+
+| Method | Path                 |
+| :----- | :------------------- |
+| `POST` | `/sys/quotas/config` |
+
+### Parameters
+
+- `enable_rate_limit_audit_logging` `(bool: false)` - If set, starts audit logging
+  of requests that get rejected due to rate limit quota rule violations.
+
+### Sample Payload
+
+```json
+{
+  "enable_rate_limit_audit_logging": true,
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request POST \
```

Comment:
I believe this endpoint is an authenticated endpoint that it requires a client token.

```suggestion
    --request POST \
    --header "X-Vault-Token: ..." \
```

Unless the code has changed recently.